contracts/RMRK/utils/RMRKNestableUtils.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.18;
+
+import "../nestable/IERC6059.sol";
+import "../library/RMRKErrors.sol";
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+import "hardhat/console.sol";
+
+/**
+ * @title RMRKNestableUtils
+ * @author RMRK team
+ * @notice Smart contract of the RMRK Nestable  utils module.
+ */
+contract RMRKNestableUtils {
+    /**
+     * @notice Used to validate whether the specified child token is owned by a given parent token.
+     * @param parentAddress Address of the parent token's collection smart contract
+     * @param childAddress Address of the child token's collection smart contract
+     * @param parentId ID of the parent token
+     * @param childId ID of the child token
+     * @return A boolean value indicating whether the child token is owned by the parent token or not
+     */
+    function validateChildOf(
+        address parentAddress,
+        address childAddress,
+        uint256 parentId,
+        uint256 childId
+    ) public view returns (bool) {
+        if (
+            !IERC165(childAddress).supportsInterface(type(IERC6059).interfaceId)
+        ) {
+            return false;
+        }
+
+        (address directOwner, uint256 ownerId, ) = IERC6059(childAddress)
+            .directOwnerOf(childId);
+
+        if (directOwner == parentAddress && ownerId == parentId) {
```

Comment:
A detail, but you could directly return the contents of this `if`

Reply:
Right 😅 🙈